requirements/requirements-zero.txt

```diff
@@ -1 +1 @@
-deepspeed
\ No newline at end of file
+deepspeed
```

Comment:
Not relevant to logging and DeepSpeed is no longer needed.

Reply:
OK, I remove it.